src/H5TSint.c

```diff
@@ -74,6 +77,12 @@ typedef struct H5TS_tinfo_node_t {
 /* Local Prototypes */
 /********************/
 static H5TS_tinfo_node_t *H5TS__tinfo_create(void);
+#ifdef H5_HAVE_CONCURRENCY
```

Comment:
Routines to support the "disable locking for this thread" ("DLFTT") protocol when making user callbacks.